modules/lfs/filesystem_client.go

```diff
@@ -39,7 +39,7 @@ func (c *FilesystemClient) objectPath(oid string) string {
 }
 
 // Download reads the specific LFS object from the target path
-func (c *FilesystemClient) Download(ctx context.Context, objects []Pointer, callback DownloadCallback) error {
+func (c *FilesystemClient) Download(_ context.Context, objects []Pointer, callback DownloadCallback) error {
```

Comment:
The lack of use of ctx here is possibly an error and it likely should be being checked before callback is called.